tests/int/graphql/grades.graphql.test.js

```diff
@@ -0,0 +1,196 @@
+                sum_grade_w_count
+                average_gpa
+            }
+            grade_distributions {
+                grade_a_count
+                grade_b_count
+                grade_c_count
+                grade_d_count
+                grade_f_count
+                grade_p_count
+                grade_w_count
+                grade_np_count
+                average_gpa
+                course_offering {
+                  year
+                  quarter
+                  final_exam
+                  instructors
+                  max_capacity
+                  meetings {
+                    building
+                    days
+                    time
+                  }
+                  section {
+                    code
+                    comment
+                    number
+                    type
+                  }
+                  course {
+                      id
+                      department
+                      number
+                  }
+              }
+            }
+        }
+    }`})
+  .set('Accept', 'application/json')
+  .expect('Content-Type', /json/)
+  .expect(200)
+  .then((response) => {
+      expect(response.body).toHaveProperty('data');
+      expect(response.body["data"]).toHaveProperty('grades');
+      expect(response.body["data"]["grades"]["aggregate"]).toEqual(
+          expect.objectContaining({
+              "sum_grade_a_count": expect.any(Number),
+              "sum_grade_b_count": expect.any(Number),
+              "average_gpa": expect.any(Number)})
+      );
+      expect(response.body["data"]["grades"]["aggregate"]["average_gpa"]).toBeLessThanOrEqual(4.0);
+      expect(Array.isArray(response.body["data"]["grades"]["grade_distributions"])).toBeTruthy();
+      expect(response.body["data"]["grades"]["grade_distributions"][0]).toEqual(
+        expect.objectContaining({
+            "grade_a_count": expect.any(Number),
+            "grade_b_count": expect.any(Number),
+            "average_gpa": expect.any(Number)
+        })
+      );
+      expect(response.body["data"]["grades"]["grade_distributions"][0]["average_gpa"]).toBeLessThanOrEqual(4.0);
+      expect(response.body["data"]["grades"]["grade_distributions"][0]["course_offering"]).toEqual(
+          expect.objectContaining({
+              "year": "2019-20",
+              "quarter": expect.any(String),
+              "section": expect.any(Object)
+          })
+      );
+      expect(response.body["data"]["grades"]["grade_distributions"][0]["course_offering"]["course"]).toEqual(
+        expect.objectContaining({
+            "department": "I&C SCI",
+            "number": "33",
+        })
+    );
+    
+  }));
+});
+
+
+describe('POST /graphql/', () => {
+  it('GraphQL: Grades Query, multiple years',  () => request
+  .post('/graphql/')
+  .send({query:`{
+    grades(year:"2019-20;2018-19", department: "I&C SCI") {
+        grade_distributions {
+          	course_offering {
```

Comment:
```suggestion
          course_offering {
```
Nit: fix indentation

Reply:
This is weird. It looks fine on my end, but I do think I've fixed it.